docs/operations/migration_policies.md

```diff
@@ -0,0 +1,161 @@
+# Migration Policies
+
+Migration policies provides a new way of applying migration configurations to Virtual Machines. The policies
+can refine Kubevirt CR's `MigrationConfiguration` that sets the cluster-wide migration configurations. This way,
+the cluster-wide settings serve as a default while can be changed, removed or added by the migration policy.
+
+## Overview
+
+KubeVirt supports [Live Migrations](./live_migration.md) of Virtual Machine workloads.
+Before migration policies were introduced, migration settings could be configurable only on the cluster-wide
+scope by editing [KubevirtCR's spec](https://kubevirt.io/api-reference/master/definitions.html#_v1_kubevirtspec)
+or more specifically [MigrationConfiguration](https://kubevirt.io/api-reference/master/definitions.html#_v1_migrationconfiguration)
+CRD.
+
+Several aspects (although not all) of migration behaviour that can be customized are:
+- Bandwidth
+- Auto-convergence
+- Post/Pre-copy
+- Max number of parallel migrations
+- Timeout
```

Comment:
Where are these items explained better?

We need to explain them /somewhere/ reallly well.

Reply:
They're briefly explained here: https://kubevirt.io/user-guide/operations/live_migration/#changing-cluster-wide-migration-limits

But I fully agree, we need to improve the explanation for sure.

Reply:
+ explain that you can refine these with migration policies